### PR TITLE
add image apis

### PR DESCRIPTION
**Pull request by festinalente91**
- add s3 upload api
- add google vision ocr api

### CODE REVIEW

File: app/utils/images.util.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        ]
+      }]
+    };
+
+    return ImageAPIUtil.encodeBase64(filePath)
+      .then(data => {
+        body.requests[0].image.content = data;
+        return fetch(address, {
+          method: 'POST',
+          'content-type': 'application/json',
```
Comment (ochanje210) [inline app/utils/images.util.js]: shouldn't this be in headers?

Comment (festinalente91) [reply app/utils/images.util.js]: right!

**Comment by ochanje210** [comment]
overall looks good

**Comment by ace0625** [comment]
Please resolve the conflicts @festinalente91

**Pull request closed without merging**